### DISCUSSION TITLE
Manual sorting of cards in Kanban view

### DISCUSSION THREAD

**Discussion by Stefan-59** [marcusolsson/obsidian-projects#706] [Help]
First of all: thank you for the great plugin! 

When in Kanban view, I'd like to sort the Cards / Notes in a column manually. I.E. I have a column 'next actions' and I want to have the next one I have to work on at the top of the list.
When dragging the card it LOOKS as if it would be sorted (the other card makes room where I seem to be able to drop the new card), but as soon as I drop the card, all jumps back to its former order.

Am I doing something wrong, or is this some kind of 'feature' ;-) 

EDIT: I found some older posts dealing with the same question. Is there any news about the plans to re-implement the manual reorder-feature?

**Comment by Acylation** [comment]
We welcome back this feature in the latest release v1.17.2, check PR #788 for the details!

**Reply by tosascott** [reply]
@Stefan-59, going beyond manually arranging the column order, it is possible to configure the status field to have a pre-defined set of values. Once you do that, the columns will be retained even if empty. I put a _How To_ in https://github.com/marcusolsson/obsidian-projects/issues/653#issuecomment-2028804463
